roles/vgcn.config/tasks/main.yml

```diff
@@ -1,24 +1,36 @@
 ---
 - name: Create directories for configuration files
   file:
```

Comment:
Maybe it makes sense to specify a mode here?
for the dirs we are creating here it would be all `755`
I forgot that we also need to add the public keys to the users, and therefore need to create
in a `home/centos` dir:
~~~
drwx------ 1 1000 1000  30 May  7 13:44 .
drwxr-xr-x 1 1000 1000  8 May  7 13:40 ..
-rw------- 1 1000 1000 736 May  7 13:42 authorized_keys

~~~